packages/connectors/connector-wecom/package.json

```diff
@@ -14,7 +14,8 @@
   "files": [
     "lib",
     "docs",
-    "logo.svg"
+    "logo.svg",
+    "logo-dark.svg"
```

Comment:
Is this an expected change in this PR?

Reply:
This was fixed by @wangsijie  in a previous PR, should rebase to eliminate this.

Reply:
rebase will resolve this issue.